apps/src/blocklyAddons/cdoBlockSvg.js

```diff
@@ -5,16 +5,6 @@ export default class BlockSvg extends GoogleBlockly.BlockSvg {
   constructor(workspace, prototypeName, opt_id) {
     super(workspace, prototypeName, ++Blockly.uidCounter_); // Use counter instead of randomly generated IDs
 
-    /* *
-     * Workaround to make block id available on SVG in IE.
-     * I opened https://github.com/google/blockly/issues/4419 so hopefully we can remove this with the next
```

Comment:
https://github.com/google/blockly/pull/4425 is included in https://github.com/google/blockly/releases/tag/4.20201217.0 so we can remove this (I also double-checked in Saucelabs that this is fine to remove)